common/s3/s3_client.go

```diff
@@ -42,35 +43,6 @@ type S3Client interface {
 
 	// CreateBucket creates a bucket in S3.
 	CreateBucket(ctx context.Context, bucket string) error
-
-	// FragmentedUploadObject uploads a file to S3. The fragmentSize parameter specifies the maximum size of each
-	// file uploaded to S3. If the file is larger than fragmentSize then it will be broken into
-	// smaller parts and uploaded in parallel. The file will be reassembled on download.
-	//
-	// Note: if a file is uploaded with this method, only the FragmentedDownloadObject method should be used to
-	// download the file. It is not advised to use DeleteObject on files uploaded with this method (if such
-	// functionality is required, a new method to do so should be added to this interface).
-	//
-	// Note: if this operation fails partway through, some file fragments may have made it to S3 and others may not.
-	// In order to prevent long term accumulation of fragments, it is suggested to use this method in conjunction with
-	// a bucket configured to have a TTL.
-	FragmentedUploadObject(
```

Comment:
`FragmentedUploadObject` is gone, but it's still mentioned by name in the doc for `ListObjects`